examples/native/native_test.c

```diff
@@ -1450,13 +1474,15 @@ int TPM2_Native_TestArgs(void* userCtx, int argc, char *argv[])
         TPM2_FlushContext(&cmdIn.flushCtx);
     }
 
+#if 1 //ndef WOLFTPM_WINAPI
```

Comment:
Consider a cleanup here... If it should always be on.

Reply:
Done